README.md

```diff
@@ -1,38 +1,82 @@
-# Newdle
+## WARNING: newdle is not yet production ready
```

Comment:
Having this above the "Newdle" title looks weird IMO

Reply:
Isn't that the point? It should be on top so that it is visible?